include/dem/find_cell_neighbors.h

```diff
@@ -69,6 +69,37 @@ class FindCellNeighbors
     typename DEM::dem_data_structures<dim>::cells_neighbor_list
       &cells_ghost_neighbor_list);
 
+  /**
+   * Finds the neighbor list (without repetition) of all the active cells in the
+   * triangulation
```

Comment:
I would use this occasion to make this brief a bit more verbose. Also please use the @brief keyword.